lib/reports/cost_report.rb

```diff
@@ -177,20 +172,6 @@ def read_freq_tables(dir, file)
       end
     end
 
-    def compile_frequency_table
```

Comment:
We no longer expect to do this in a standalone way -- we were only really using this in test; the tests are now changed to precompute a frequency table and pass it in.